products/rhel8/profiles/ism_o.profile

```diff
@@ -28,118 +28,56 @@ description: |-
-  - snmpd_use_newer_protocol
-
-  ## Wireless networks
-  ## Identifiers 1315
-  - wireless_disable_interfaces
-
-  ## ASD Approved Cryptographic Algorithms
-  ## Identifiers 0471 / 0472 / 0473 / 0474 / 0475 / 0476 / 0477 / 
-  ## 0479 / 0480 / 0481 / 0489 / 0497 / 0994 / 0998 / 1001 /  1139 / 
-  ## 1372 / 1373 / 1374 / 1375
-  - enable_fips_mode
-  - var_system_crypto_policy=fips
-  - configure_crypto_policy
-
-  ## Secure Shell access
-  ## Identifiers 0484 / 1506 / 1449 / 0487
-  - sshd_allow_only_protocol2
-  - sshd_enable_warning_banner
-  - sshd_disable_x11_forwarding
-  - file_permissions_sshd_private_key
+    - ism_o:all
+    # Add rules missing from the control file that where in RHEL 8
+    - accounts_password_all_shadowed
+    - usbguard_allow_hid_and_hub
+    - sshd_allow_only_protocol2
+    # Remove rules that where not the original profile for RHEL 8
+    - '!accounts_password_minlen_login_defs'
+    - '!accounts_password_pam_dcredit'
+    - '!accounts_password_pam_lcredit'
+    - '!accounts_password_pam_minclass'
+    - '!accounts_password_pam_ocredit'
+    - '!accounts_password_pam_ucredit'
+    - '!accounts_passwords_pam_tally2_deny_root'
+    - '!accounts_passwords_pam_tally2_unlock_time'
+    - '!audit_access_failed_aarch64'
+    - '!audit_access_failed_ppc64le'
+    - '!audit_access_success_aarch64'
+    - '!audit_access_success_ppc64le'
+    - '!audit_rules_unsuccessful_file_modification_creat'
+    - '!audit_rules_unsuccessful_file_modification_ftruncate'
+    - '!audit_rules_unsuccessful_file_modification_open'
+    - '!audit_rules_unsuccessful_file_modification_openat'
+    - '!audit_rules_unsuccessful_file_modification_open_by_handle_at'
+    - '!audit_rules_unsuccessful_file_modification_truncate'
+    - '!chronyd_configure_pool_and_server'
+    - '!configure_kerberos_crypto_policy'
+    - '!configure_opensc_card_drivers'
+    - '!dnf-automatic_apply_updates'
+    - '!enable_dracut_fips_module'
+    - '!enable_ldap_client'
+    - '!force_opensc_card_drivers'
+    - '!kerberos_disable_no_keytab'
+    - '!network_ipv6_static_address'
+    - '!package_audit_installed'
+    - '!package_libdnf-plugin-subscription-manager_installed'
+    - '!package_opensc_installed'
+    - '!package_pcsc-lite-ccid_installed'
+    - '!package_pcsc-lite_installed'
+    - '!package_subscription-manager_installed'
+    - '!sebool_authlogin_nsswitch_use_ldap'
+    - '!sebool_authlogin_radius'
+    - '!sebool_kerberos_enabled'
+    - '!secure_boot_enabled'
+    - '!service_pcscd_enabled'
+    - '!set_password_hashing_algorithm_libuserconf'
+    - '!set_password_hashing_algorithm_logindefs'
+    - '!set_password_hashing_algorithm_passwordauth'
+    - '!set_password_hashing_algorithm_systemauth'
+    - '!sssd_enable_smartcards'
+    - '!system_booted_in_fips_mode'
+    # Adjust variables to match the origianl RHEL 8 profiles
+    - var_password_hashing_algorithm_pam=sha512
+    - var_accounts_password_minlen_login_defs=default
```

Comment:
I have a vague memory that you cannot select the `default` value as it's not really defined in the datastream, in this case you can simply select `15`.

```suggestion
    - var_accounts_password_minlen_login_defs=15
```

Reply:
I will change it. But we should have a CI and docs change for that.

Reply:
Yeah, so here is the actual explanation for the `default` selector:

https://github.com/ComplianceAsCode/content/issues/7222#issuecomment-877245366